lib/classes/class-settings.php

```diff
@@ -127,6 +127,48 @@ public function __construct($args = false) {
           }
           $this->set('property_types', $ar);
         }
+        
+        /* Standard Attributes 
+         * added 28/07/2016 @raj
+         * 
+         */
+
+        $PDF_unique = array(
```

Comment:
@raj-uimatic I know, I suggested to use `/static/config/default.settings.json`. But, let's move it to another place.

We have `composer.json` in root directory. Please, move json from `standard_attributes.json` to `extra.schemas.standard_attributes` in composer.json.

The main reason is to have one place where we store all json settings. Also, you have to localize this data.

Use the following method for getting and localize standard attributes:

https://github.com/wp-property/wp-property/blob/feature-standard-attribute/vendor/libraries/usabilitydynamics/lib-wp-bootstrap/lib/classes/class-bootstrap.php#L140-L174


User Singleton for it:

`ud_get_wp_property()->get_schema( 'use.your.path.here' )`

For localization, see:

https://github.com/wp-property/wp-property/blob/feature-standard-attribute/vendor/libraries/usabilitydynamics/lib-wp-bootstrap/lib/classes/class-bootstrap.php#L176-L195

AND

https://github.com/wp-property/wp-property/blob/feature-standard-attribute/vendor/libraries/usabilitydynamics/lib-wp-bootstrap/lib/classes/class-bootstrap.php#L176-L195

Let me know if you have questions